packages/pyright-internal/src/localization/simplified.nls.de.json

```diff
@@ -0,0 +1,93 @@
+        "expectedBinaryRightHandExpr": "Auf der rechten Seite vom Operator wird ein Ausdruck erwartet",
+        "expectedClassName": "Klassenname fehlt",
+        "expectedCloseBrace": "Schließende geschweifte Klammer „}“ fehlt.",
+        "expectedCloseBracket": "Schließende eckige Klammer „]“ fehlt.",
+        "expectedCloseParen": "Schließende runde Klammer „)“ fehlt.",
+        "expectedColon": "Doppelpunkt „:“ fehlt",
+        "expectedDelExpr": "Ausdruck nach „del“ fehlt.",
+        "expectedElse": "„else“ fehlt",
+        "expectedEqualityOperator": "Hier wird ein Gleichheitsoperator erwartet, meinten Sie \"==\"?",
+        "expectedExpr": "Ausdruck fehlt",
+        "expectedFunctionName": "Funktionsname nach \"def\" fehlt.",
+        "expectedIdentifier": "Bezeichner fehlt",
+        "expectedImport": "„import“ fehlt",
+        "expectedImportAlias": "Fehlendes Symbol nach \"as\"",
+        "expectedImportSymbols": "Nach dem Import wird ein oder mehrere Symbolnamen erwartet",
+        "expectedIn": "„in“ fehlt.",
+        "expectedIndentedBlock": "Einrückung fehlt",
+        "expectedInExpr": "Fehlender Ausdruck nach \"in\"",
+        "expectedMemberName": "Fehlender Mitgliedsname nach \".\"",
+        "expectedModuleName": "Modulname fehlt",
+        "expectedNamedParameter": "Schlüsselwortparameter muss \"*\" folgen",
+        "expectedNewlineOrSemicolon": "Unerwartete zusätzliche Inhalte\nEs wird eine Anweisung pro Zeile erwartet",
+        "expectedOpenParen": "Öffnende runde Klammer „)“ fehlt.",
+        "expectedParamName": "Fehlender Parametername",
+        "expectedSliceIndex": "Fehlender Index oder Slice",
+        "expectedYieldExpr": "Ein Ausdruck wird in der yield-Anweisung erwartet",
+        "globalReassignment": "\"{name}\" wird vor der globalen Deklaration zugewiesen",
+        "globalRedefinition": "\"{name}\" wurde bereits als global deklariert",
+        "importResolveFailure": "Modul „{importName}“ konnte nicht gefunden werden.",
+        "importSymbolUnknown": "„{name}“ konnte im Modul „{moduleName}“ nicht gefunden werden.",
+        "inconsistentIndent": "Einrückung stimmt nicht mit der vorherigen Zeile überein",
+        "inconsistentTabs": "Inkonsistente Verwendung von Tabs und Leerzeichen für die Einrückung",
+        "instanceMethodSelfParam": "Instanzmethoden benötigen einen \"self\"-Parameter",
+        "invalidIdentifierChar": "Ungültiges Zeichen im Bezeichner",
+        "invalidTokenChars": "Ungültiges Zeichen \"{text}\" im Token",
+        "memberAccess": "Kann nicht auf Mitglied „{name}“ für Typ „{type}“ zugreifen.",
+        "memberDelete": "Kann Mitglied \"{name}\"für Typ \"{type}\" nicht löschen",
+        "memberSet": "Kann Mitglied \"{name}\" nicht für Typ \"{type}\" zuweisen",
+        "moduleNotCallable": "Modul ist nicht abrufbar.",
+        "moduleUnknownMember": "„{name}“ ist kein bekanntes Mitglied des Moduls „{module}“.",
```

Comment:
There's a few unexpected full stops, mostly towards the end of the file.

Reply:
I may as well fix these

Reply:
Raised with Emma as they've been introduced recently